src/main/resources/db/changelog/v1.0/changes/2024_08_21_add_book_table.yaml

```diff
@@ -0,0 +1,13 @@
+databaseChangeLog:
+  - changeSet:
+      id: 2024_08_21_add_book_table
+      author: andrii
+      dbms: mysql,h2
+      changes:
+        - sqlFile:
+            dbms: mysql
+            encoding: utf8
+            path: ../data/2024-08-21_add_book_table.sql

```

Comment:
yaml script format is considered more flexible and recommended

Reply:
I've used this format in my previous project and it was a wonderful experience. Try it.